games/s.yaml

```diff
@@ -162,6 +162,20 @@
   license: [As-is]
   remakes: [Shining Soul]
 
+- name: Shockolate
+  updated: 2018-06-15
+  repo: https://github.com/Interrupt/systemshock
+  development: active
+  status: playable
+  lang: [C]
+  framework: [SDL]
```

Comment:
should be SDL2

Reply:
I wasn't sure if frameworks are versioned and the readme didn't say. Fixed.